web/beacon-app/src/features/auth/hooks/useDisplayToast.ts

```diff
@@ -0,0 +1,38 @@
+import { t } from '@lingui/macro';
+import { useEffect } from 'react';
+import toast from 'react-hot-toast';
+
+import { updateQueryStringValueWithoutNavigation } from '@/utils/misc';
+const useDisplayToast = (param: any) => {
+  // handle toast from successfull email verification
+  useEffect(() => {
+    if (param?.accountVerified && param?.accountVerified === '1') {
+      const isVerified = localStorage.getItem('isEmailVerified');
+      if (isVerified === 'true') {
+        toast.success(
+          t`Thank you for verifying your email address.
+          Log in now to start using Ensign.`
+        );
+      }
+    }
+    return () => {
+      localStorage.removeItem('isEmailVerified');
+    };
+  }, [param?.accountVerified]);
+
+  // handle toast from successfull reset password
+
+  useEffect(() => {
+    if (param?.from && param?.from === 'reset-password') {
+      toast.success(
+        t`Your password has been reset successfully. Please log in with your new password.`
```

Comment:
After completing the reset password workflow, I saw this toast message twice. Here's a link to a video recording: https://www.awesomescreenshot.com/video/21279971?key=a083bbd62e4bdff33e21fba59ff19d8c